robyn/__init__.py

```diff
@@ -30,15 +27,23 @@ def __init__(self, file_object: str) -> None:
         self.file_path = file_object
         self.directory_path = directory_path
         self.config = Config()
```

Comment:
Hey @AntoineRR ,

I have an unrelated suggestion to the PR. But this will help us with writing integration tests for the dev server.

We can pass the config as a part of the Robyn initializer. 

```
def __init__(self, file_object: str, config: Config = Config()):
...
  self.config = config
```

This way we will be able to pass in a dev flag and custom flag while writing test code.

Reply:
That's a good idea, I added this :smile: